--- detector.rs	2020-06-15 10:40:49.000000000 -0400
+++ detector_new.rs"	2020-06-30 10:56:31.931406500 -0400
@@ -46,6 +46,7 @@
     /// Resolves the reference against a local file path
     ///
     /// This is similar to `resolve` but operates on file paths.
+    #[cfg(any(unix, windows, target_os = "redox"))]
     pub fn resolve_path(&self, minified_path: &Path) -> Option<PathBuf> {
         let url = self.get_url();
         if url.starts_with("data:") {
